src/presenter/QuestionsListPresenter.js

```diff
@@ -0,0 +1,58 @@
+import model from "../model/model";
+
+class QuestionsListPresenter {
+    onCreateQuestion(){
+       window.location.assign("#/create-question");
+
+    }
+
+    onChangeQuestion(property, value){
+        model.changeNewQuestionProperty(property, value);
+    }
+
+    onFilterByTitleQuestion(){
+        model.clearFilters();
+
+        let iterator;
+        for(iterator=0; iterator<model.state.questions.length; iterator++){
```

Comment:
It would have been better to do the filtering in the model (via a method). 

You could also do it more easily, by using the `filter` function: `model.state.questions.filter(question => question.title === model.state.toFilter)`